src/Microsoft.AspNetCore.Server.IIS/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
I don't like `InternalsVisibleTo`, is it really necessary here?

Reply:
Right now, the [IISHttpContext is internal](https://github.com/aspnet/IISIntegration/blob/release/2.2/src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.cs#L25). We would need to expose a lot to remove the InternalsVisibleTo.

Reply:
Why do you need  IISHttpContext in test?

Reply:
The tests needs IISEventSource, which uses IISHttpContext https://github.com/aspnet/IISIntegration/pull/1374/files#diff-9e79b380478e1d8cf32c8b5944147c9eR19

Reply:
I think it's fine to use IVT here considering it's exactly what we do in kestrel https://github.com/aspnet/KestrelHttpServer/blob/release/2.2/src/Kestrel.Core/Properties/AssemblyInfo.cs